bin/kano-os-recovery

```diff
@@ -44,13 +44,18 @@ if [ $system_stable != 1 ] && [ "$mode" == "--recovery" ]; then
     /bin/mount /dev/mmcblk0p1 /boot
     /bin/mount -o remount,rw /dev/mmcblk0p2 /
 
+    # We do not want to start init.d services during the recovery.
+    # Several packages have this assumption on their postinst - dnsmasq, dhcpcd5
+    /bin/mount --bind /bin/true /usr/sbin/invoke-rc.d
```

Comment:
NB this was my ugly suggestion, sorry guys. Can't think of a better way -
except to add `MountAPIVFS=true` to the systemd service file so this mount can't affect other processes.

Reply:
Yep, not an elegant solution indeed, I'll try your suggestion @Ealdwulf and see how it goes.

Reply:
Apparently you can also do the mount in the unit file with `BindPaths=`